sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -1200,6 +1209,10 @@ class MLPRegressor(BaseMultilayerPerceptron, RegressorMixin):
     epsilon : float, optional, default 1e-8
         Value for numerical stability in adam. Only used when solver='adam'
 
+    n_iter_no_change : int, optional, default 2
```

Comment:
The default documented is wrong (as the default has changed). Can you update this?